src/include/execution/compiler/operator/aggregate_translator.h

```diff
@@ -42,6 +43,19 @@ class AggregateBottomTranslator : public OperatorTranslator {
   void Abort(FunctionBuilder *builder) override;
   void Consume(FunctionBuilder *builder) override;
 
+  /**
+   * @return The pipeline work function parameters.
+   */
+  util::RegionVector<ast::FieldDecl *> GetWorkerParams() override { UNREACHABLE("Not implemented yet"); }
```

Comment:
Can this be abstracted into an interface? Like the `GetWorkerParams` is by default unreachable?

Reply:
We were also talking about this but since we were told that others will work on these operators during the summer, we decided to have these signatures here "to be implemented"